pkg/network/tracer/connection/fentry/probes.go

```diff
@@ -101,6 +105,12 @@ func enableProgram(enabled map[string]string, name string) {
 func enabledPrograms(c *config.Config) (map[string]string, error) {
 	enabled := make(map[string]string, 0)
 	if c.CollectTCPConns {
+		if c.ClassificationSupported() {
+			enableProgram(enabled, probes.ProtocolClassifierEntrySocketFilter)
+			enableProgram(enabled, string(probes.ProtocolClassifierSocketFilter))
```

Comment:
Is the `string` casting needed here? Did not see it in pkg/network/tracer/connection/kprobe/config.go

Reply:
No it shouldn't be. Likely result of a bad merge. I'll fix it.